client/my-sites/site-settings/section-import.jsx

```diff
@@ -69,21 +66,25 @@ export default React.createClass( {
 		}
 
 		return status.map( item => Object.assign( {}, item, { site, siteTitle } ) );
-	},
+	}
 
-	updateFromAPI: function() {
+	updateFromAPI = () => {
 		fetchState( this.props.site.ID );
-	},
+	};
```

Comment:
Though I have a personal preference for not using semis there (we're already closing a block, so where's the ambiguity?), the logic is that we're not really defining a method anymore, but rather making an assignment of a bound method:

```js
updateFromAPI = expr;
```

Where `expr` _happens_ to be a function expression.